tools/azure-pipelines/jobs-template.yml

```diff
@@ -163,6 +163,7 @@ jobs:
       IT_CASE_S3_SECRET_KEY: $(SECRET_S3_SECRET_KEY)
 
   - task: PublishTestResults@2
+    condition: succeededOrFailed()
```

Comment:
what happens if say, the compilation fails? Is this case handle gracefully?

Reply:
The `PublishTestResults` will produce a warning that no tests were found.
But we actually have this warning in every run (because the python profile has no Junit tests: https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=2125&view=results